frontend/src/Editor/Components/PDF.jsx

```diff
@@ -47,46 +51,107 @@ export const PDF = React.memo(({ styles, properties, width, height }) => {
     [pageNumber]
   );
 
+  // styles for download icon
+  const downloadIconOuterWrapperStyles = {
+    backgroundColor: 'white',
+    borderRadius: '4px',
+    height: '36px',
+    padding: '0.5rem',
+    cursor: 'pointer',
+  };
+  const downloadIconImgStyle = {
+    width: '15px',
+    height: '15px',
+  };
   const renderPDF = () => (
-    <Document file={url} onLoadSuccess={onDocumentLoadSuccess} className="pdf-document">
-      {Array.from(new Array(numPages), (el, index) => (
-        <Page
-          pageNumber={index + 1}
-          width={scale ? width - 12 : undefined}
-          height={scale ? undefined : height}
-          key={`page_${index + 1}`}
-          inputRef={(el) => (pageRef.current[index] = el)}
-        />
-      ))}
-      {pageControls && (
-        <>
-          <div className="pdf-page-controls">
-            <button disabled={pageNumber <= 1} onClick={() => updatePage(-1)} type="button" aria-label="Previous page">
-              ‹
-            </button>
-            <span>
-              {pageNumber} of {numPages}
-            </span>
-            <button
-              disabled={pageNumber >= numPages}
-              onClick={() => updatePage(1)}
-              type="button"
-              aria-label="Next page"
-            >
-              ›
-            </button>
-          </div>
-        </>
-      )}
-    </Document>
+    <>
+      <Document
+        file={url}
+        onLoadSuccess={onDocumentLoadSuccess}
+        onLoadError={onDocumentLoadError}
+        className="pdf-document"
+      >
+        {Array.from(new Array(numPages), (el, index) => (
+          <Page
+            pageNumber={index + 1}
+            width={scale ? width - 12 : undefined}
+            height={scale ? undefined : height}
+            key={`page_${index + 1}`}
+            inputRef={(el) => (pageRef.current[index] = el)}
+          />
+        ))}
+      </Document>
+    </>
```

Comment:
```suggestion
      <Document
        file={url}
        onLoadSuccess={onDocumentLoadSuccess}
        onLoadError={onDocumentLoadError}
        className="pdf-document"
      >
        {Array.from(new Array(numPages), (el, index) => (
          <Page
            pageNumber={index + 1}
            width={scale ? width - 12 : undefined}
            height={scale ? undefined : height}
            key={`page_${index + 1}`}
            inputRef={(el) => (pageRef.current[index] = el)}
          />
        ))}
      </Document>
```

We can remove the fragments as we already have a common parent